recipe/meta.yaml

```diff
@@ -16,26 +16,34 @@ build:
 
 requirements:
   build:
+    - {{ stdlib("c") }}
     - {{ compiler('cxx') }}
     - cmake
     - make
-  run_constrained:
-    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx]
+  host:
+    - zlib
+    - bzip2
 
 test:
   source_files:
     - source/test
   requires:
+    - {{ stdlib("c") }}
     - {{ compiler('cxx') }}
     - {{ compiler('fortran') }}
+    - coreutils  # [osx]
   commands:
-    # Workaround for lack of CONDA_BUILD_SYSROOT variable in staged recipes
-    - export CONDA_BUILD_SYSROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk  # [osx]
+    # Ensure 'g' prefixed Unix commands available for macOS
+    - brew install coreutils  # [osx]
```

Comment:
```
brew: command not found
```

Okay that didn't work. So as mentioned in https://github.com/conda-forge/hepmc3-feedstock/pull/19#discussion_r1727783965 we need a way to get the equivalent of the `brew install coreutils` 'g' prefixed utils for macOS. Given that `HepMC3-config` uses `greadlink` on macOS.

Reply:
Either:

* Add `coreutils` to the `build` requirements
* Patch it

Reply:
I just read more closely, please never use brew inside a conda recipe!

Reply:
> Either:
> 
> * Add coreutils to the build requirements
> * Patch it

@chrisburr From https://github.com/conda-forge/hepmc3-feedstock/pull/19#discussion_r1727797883 it seems that https://github.com/conda-forge/coreutils-feedstock/ provides GNU coreutils with the normal naming. It won't provide the 'g' prefixed versions that macOS wants.

So seems that we'll need to patch https://gitlab.cern.ch/hepmc/HepMC3/-/blob/254c5f4fb9d7a3d7c6c04f3508d5cbbaeac6e707/HepMC3-config.in#L7-8

> I just read more closely, please never use brew inside a conda recipe!

Okay good. This is what I assumed, but the suggestion to check if this worked was more a desperation check. Sorry to have caused any cortisol levels to rise earlier than hoped for today. :grimacing: